docs/environment.md

```diff
@@ -1,76 +1,119 @@
-# Setting up your environment
+# Set up InnerEye-DeepLearning
 
-## Prerequisites
+## Operating System
 
-In order to work with the solution, your operating system environment will need [git](https://git-scm.com/) and [git lfs](https://git-lfs.github.com/) installed. Depending on the operating system that you are running the installation instructions may vary. Please refer to respective documentation sections on the tools' websites for detailed instructions.
+We recommend using our toolbox with [Ubuntu 20.04 LTS](https://releases.ubuntu.com/20.04/).  Most core InnerEye functionality will be stable on other operating systems, but PyTorch's full feature set is only available on Linux. All jobs in AzureML, both training and inference, run from an Ubuntu 20.04 Docker image. This means that using Ubuntu 20.04 locally allows for maximum reproducibility between your local and AzureML environmnets.
 
-We recommend using PyCharm or VSCode as the Python editor.
+For Windows users, Ubuntu can be set up with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). Please refer to the [InneryEye WSL docs](docs/WSL.md) for more detailed instructions on getting WSL set up.
 
-You have two options for working with our codebase:
+MacOS users can access an Ubuntu OS through [VirtualBox](https://www.virtualbox.org/wiki/Downloads).
 
-* You can fork the InnerEye-DeepLearning repository, and work off that. We recommend that because it is easiest to set up.
-* Or you can create your project that uses the InnerEye-DeepLearning code, and include InnerEye-DeepLearning as a git
-submodule. We only recommended that if you are very handy with Python. More details about this option
-[are here](innereye_as_submodule.md).
+## Clone Repository
 
-### Set Up Windows Subsystem for Linux
+1. Ensure you have [Git CLI](https://git-scm.com/downloads) installed.
+2. Install Git Large File Storage (LFS):
 
-When developing on a Windows machine, we recommend using [the Windows Subsystem for Linux, WSL2](https://docs.microsoft.com/en-us/windows/wsl/about).
-That's because PyTorch has better support for Linux.  If you want to use WSL2, please follow
-[these instructions](/docs/WSL.md) , that correspond to the manual installation in the official docs.
+   ```shell
+   git lfs install
+   ```
+
+3. Clone the repository:
+
+   ```shell
+   git clone --recursive https://github.com/microsoft/InnerEye-DeepLearning
+   cd InnerEye-DeepLearning
+   ```
+
+To view and edit the InnerEye code, we recommended using the [VSCode](https://code.visualstudio.com/) IDE.
+
+## Set Up Conda
+
+[Conda](https://docs.conda.io/en/latest/) is an open source package management system. It is used in InnerEye to manage all python packages. Follow the instructions in this section to get it set up on your machine.
```

Comment:
why are we talking about Conda here and again in line 51? Maybe first do the section on build tools, then do all things Conda.

Reply:
The build tools is a prerequisite to Conda so I thought it made sense to introduce Conda first and then go through the relevant steps to getting it installed. Have updated the headers to make this a little clearer.